index.js

```diff
@@ -4,7 +4,40 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() 
+{
+    console.log(process.argv)
+
+    if(process.argv[2] === undefined) // checking if no operation is provided
+    {
```

Comment:
The proper formatting for the opening bracket of the if is on the same line as the if statement. The same goes for loops and else blocks.

If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?

Reply:
Hello @Angiepmunoz , 

By adding 
```
else if (!(isNaN(process.argv[2]))){
        return "Missing Operation"
    }
```
Invalid operation: 1 problem will get resolved. 

Now if you run node run.js 1 2 3, you will receive a message "Missing operation"

I also fixed the formatting of opening paranthesis

@abbreviatedman helped me with figuring this result.

Thank you Colin and Angie